tests/trilinos/sparse_matrix_copy_from_02.cc

```diff
@@ -0,0 +1,144 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+
+// Test TrilinosWrappers::SparseMatrix::add(factor, other_matrix)
+// for the case of a non-contiguous set of rows.
```

Comment:
please extend/adjust the comment so that it's clear what's the difference to the first one.